submodules/bastion/variables.tf

```diff
@@ -36,7 +36,6 @@ variable "public_subnet_id" {
 }
 
 variable "security_group_rules" {
-
   description = "Bastion host security group rules."
   type = map(object({
```

Comment:
Isnt the object missing keys
```hcl
      protocol                 = "tcp"
      from_port                = "22"
      to_port                  = "22"
      type                     = "ingress"
      description              = "Inbound ssh"
      cidr_blocks              = var.bastion.authorized_ssh_ip_ranges
      source_security_group_id = null
```

Reply:
Not sure I follow, the type is defined with all those fields. Right now the default is passed in through the main.tf module, though maybe it should be in here?